web/app.fluidity.money/app/routes/$network/query/volumeStats.tsx

```diff
@@ -22,13 +21,9 @@ export const loader: LoaderFunction = async ({ params, request }) => {
 
   // Postprocess res
   const fdaiPostprocess = (volume: Volume) => {
-    const bn = new BN(volume.amount);
-    const decimals = new BN(10).pow(new BN(12));
-    const amount = bn.div(decimals).toNumber();
-
     return {
       ...volume,
-      amount: amount,
+      amount: volume.amount / 10 ** 12,
```

Comment:
any reason for changing this? seems like it could break very easily esp since dai has so many decimals

Reply:
not rly, just wanted decimal number. i'll remove it